cmd/glasskube/cmd/update.go

```diff
@@ -117,9 +119,54 @@ func completeInstalledPackageNames(
 	return
 }
 
+func completeUpgradablePackageVersions(
+	cmd *cobra.Command,
+	args []string,
+	toComplete string,
+) ([]string, cobra.ShellCompDirective) {
+
+	var dir cobra.ShellCompDirective
```

Comment:
I think it would be good to initialize dir with `cobra.ShellCompDirectiveNoFileComp`.

Reply:
Here, There is no such `cobra.ShellCompDirectiveNoFileComp`

Reply:
Mabye your IDE only completes type names? `dir := cobra.ShellCompDirectiveNoFileComp` works for me